compiler/rustc_ast_pretty/src/pprust.rs

```diff
@@ -1670,6 +1670,15 @@ impl<'a> State<'a> {
         self.end();
     }
 
+    fn print_expr_anon_const(&mut self, expr: &ast::AnonConst, attrs: &[ast::Attribute]) {
+        self.ibox(INDENT_UNIT);
+        self.s.word("const {");
```

Comment:
The `expr` is already a block, printing the braces again shouldn't be necessary.
(Same applies to patterns.)